roles/docker_storage_to_overlay2/vars/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+storage_setup_path: "/etc/sysconfig/docker-storage-setup"
+
+# List of services to stop and restart.  Any available services
+# in optional_service_names get appended to this list beforehand.
+service_names:
+- "docker.service"
+
+optional_service_names:
+- "atomic-openshift-node.service"
```

Comment:
could you also add container-engine here?  It should be the only required change to get this playbook working with the Docker system container

I think docker.service as well should be optional since it is not used with container-engine

Reply:
@giuseppe  Will do.

BTW, the other role here uses `docker info` to fish out the current storage driver.  Is there some way to do this with `container-engine`?

```
# docker info
...
Storage Driver: overlay2
...
```